CHANGELOG.md

```diff
@@ -6,12 +6,14 @@
   * `countryCodeAlpha2` now returns a 2 character country code instead of a 3 character country code 
   * BraintreePayPalNativeCheckout
     * Remove entire PayPal Native Checkout module
-  * BraintreeVenmo
-    * Update `BTVenmoRequest` to make all properties accessible on the initializer only vs via the dot syntax.
-  * BraintreeSEPADirectDebit
-    * Update `BTSEPADirectDebitRequest` to make all properties accessible on the initializer only vs via the dot syntax.
-  * BraintreeLocalPayment
-    * Update `BTLocalPaymentRequest` to make all properties accessible on the initializer only vs via the dot syntax.
+* BraintreeVenmo
+  * Update `BTVenmoRequest` to make all properties accessible on the initializer only vs via the dot syntax.
+* BraintreeSEPADirectDebit
+  * Update `BTSEPADirectDebitRequest` to make all properties accessible on the initializer only vs via the dot syntax.
+* BraintreeLocalPayment
+  * Update `BTLocalPaymentRequest` to make all properties accessible on the initializer only vs via the dot syntax.
+* BraintreeThreeDSecure
+  * Update `BraintreeThreeDSecure` to make all properties accessible on the initializer only vs via the dot syntax.  
```

Comment:
Lets keep the above spacing. These are breaking changes so we want them to be nested under the breaking changes section. See here for similar formatting with v6: https://github.com/braintree/braintree_ios/blob/main/CHANGELOG.md#600-beta4-2023-06-01